services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/assets/[asset]/index.js

```diff
@@ -85,10 +119,10 @@ const Asset = ({ libraryData, params }) => {
   const { maintainer } = assetData.params
   const MaintainerIcon = teams[maintainer] ? teams[maintainer].pictogram : Svg64Community
 
-  const isPathAbsolute = (path) => {
+  const isPathAbsolute = (urlPath) => {
```

Comment:
Could the `createUrl` utility function you'd made be used here instead? This regex only catches *some* cases of absolute URLs, but does not catch things like `//example.com/some-route`

Reply:
Done!